pkg/apis/k0s.k0sproject.io/v1beta1/clusterconfig_types.go

```diff
@@ -71,9 +70,9 @@ type ClusterConfig struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 	metav1.TypeMeta   `json:",omitempty,inline"`
 
-	DataDir string              `json:"dataDir,omitempty"`
-	Spec    *ClusterSpec        `json:"spec,omitempty"`
-	Status  ClusterConfigStatus `json:"status,omitempty"`
+	//	DataDir string              `json:"dataDir,omitempty"`
```

Comment:
I think we can remove this line altogether.

Reply:
oh.. leftover.

I wonder what happens if there is a JSON config as CRD already. Will it not break if we remove it?

Reply:
good question. but actually adding the data dir var was a bad workaround that was never meant to be exported in the CR anyway. it is also undocumented and does not appear in default-config.

Reply:
ok lets remove it and hope it does not blow up

Reply:
I verified it. `kubectl describe   -n kube-system clusterconfig k0s` does not have anything `DataDir`.